app/livechat/server/lib/QueueManager.js

```diff
@@ -4,20 +4,26 @@ import { Match, check } from 'meteor/check';
 import { LivechatRooms, LivechatInquiry, Users } from '../../../models/server';
 import { checkServiceStatus, createLivechatRoom, createLivechatInquiry } from './Helper';
 import { callbacks } from '../../../callbacks/server';
+import { Logger } from '../../../logger';
 import { RoutingManager } from './RoutingManager';
 
+const logger = new Logger('QueueMananger');
 
 export const queueInquiry = async (room, inquiry, defaultAgent) => {
+	logger.debug(`Delegating inquiry with id ${ inquiry._id } to agent ${ defaultAgent?._id }`);
```

Comment:
I think this method should be called after:
`const inquiryAgent = RoutingManager.delegateAgent(defaultAgent, inquiry);`